### PR TITLE
Admin can update item status

### PR DESCRIPTION
**Pull request by CjMoore**
- admin can now update item status

closes #151

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -28,6 +28,8 @@ def update
     @item = Item.find(params[:id])
     @item.update(image_tag: nil) if item_params[:image]
     if @item.update_attributes(item_params)
+      @item.update_attributes(item_status: params[:item][:item_status])
```
Comment (cmacaulay) [inline app/controllers/admin/items_controller.rb:31]: 👍

File: app/views/admin/items/_form.html.erb [hunk 2]
```diff
@@ -19,6 +19,17 @@
...
+    <div class="field drop-down">
+      <%= f.label :category %>
+      <select class="browser-default" name="item[item_status]">
+        <option value="" disabled selected>Choose your status</option>
+        <option value="available">Available</option>
+        <option value="retired">Retired</option>
+      </select>
+    </div>
+  <% end %>
+
```
Comment (cmacaulay) [inline app/views/admin/items/_form.html.erb:32]: 👍

File: public/assets/application-8004f1633e0744fba70a66a1b4f57431864404bc6b3c745652f80f1d593e2bca.css [hunk 3]
```diff
@@ -16,6 +16,10 @@
 
  /* --------------- Color Glossery -------------- */
  /* # Orange darken-3: #E65100 */
+ /* # Dark Green: #042A2B */
+ /* # Minty Green(Medium Aquamarine) #69DC9E */
+ /* # Light Yellow: #F2F3AE */
+ /* # Dark Purple/eggplanty: #3A2449 */
```
Comment (cmacaulay) [inline public/assets/application-8004f1633e0744fba70a66a1b4f57431864404bc6b3c745652f80f1d593e2bca.css:22]: ❤️

**Review by cmacaulay — APPROVED** [review]
👍 Looks good! Good catch.

File: app/controllers/admin/items_controller.rb (hunk 1 above)
Comment (meyerhoferc) [reply app/controllers/admin/items_controller.rb:31]: 👍

**Review by meyerhoferc — APPROVED** [review]
Looks great, merging

**Pull request merged**